templates/bundles/PrestaShopBundle/Admin/Configure/AdvancedParameters/FeatureFlag/index.html.twig

```diff
@@ -28,43 +28,39 @@
-                <li>{{ 'In any case, you should never experiment in production.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
-              </ul>
-            </div>
-            {{ form_widget(featureFlagsForm) }}
+      <div class="card-block row">
+        <div class="card-text">
+          <div class="alert medium-alert alert-warning" role="alert">
+            {{ 'Testing a feature before its official release can be exciting. However, you must be aware of the potential risks of such experiments:'|trans({}, 'Admin.Advparameters.Notification') }}
+            <ul>
+              <li>{{ 'Experimental features are still under development. Enabling them could therefore have unintended consequences and cause data loss.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+              <li>{{ 'In any case, you should never experiment in production.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+            </ul>
           </div>
+          {{ form_widget(featureFlagsForm) }}
         </div>
-        {% if featureFlagsForm.vars.data is not empty %}
-          <div class="card-footer">
-            <div class="d-flex justify-content-end">
-              <button type="submit" id="submit-btn-feature-flag" class="btn btn-primary"
-                      data-modal-title="{{ 'Are you sure you want to enable this experimental feature?'|trans({}, 'Admin.Advparameters.Notification') }}"
-                      data-modal-message="{{ 'You are about to enable a feature that is not stable yet. This should only be done in a test environment or in full knowledge of the potential risks.'|trans({}, 'Admin.Advparameters.Notification') }}"
-                      data-modal-apply="{{ 'Enable'|trans({}, 'Admin.Actions') }}"
-                      data-modal-cancel="{{ 'Cancel'|trans({}, 'Admin.Actions') }}"
-              >{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
-            </div>
-          </div>
-        {% endif %}
       </div>
-    {% endblock %}
-    {{ form_end(featureFlagsForm) }}
-  </div>
+      {% if featureFlagsForm.vars.data is not empty %}
+        <div class="card-footer">
+          <div class="d-flex justify-content-end">
+            <button type="submit" id="submit-btn-feature-flag" class="btn btn-primary" data-modal-title="{{ 'Are you sure you want to enable this experimental feature?'|trans({}, 'Admin.Advparameters.Notification') }}" data-modal-message="{{ 'You are about to enable a feature that is not stable yet. This should only be done in a test environment or in full knowledge of the potential risks.'|trans({}, 'Admin.Advparameters.Notification') }}" data-modal-apply="{{ 'Enable'|trans({}, 'Admin.Actions') }}" data-modal-cancel="{{ 'Cancel'|trans({}, 'Admin.Actions') }}">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
```

Comment:
Is there some rule or best-practice for having inline markup versus multiline markup?

Reply:
IMO multiline markup is more readable

Reply:
Can we go back to multiline @Hlavtox ?